lib/Macros/Sources/SwiftMacros/SwiftifyImportMacro.swift

```diff
@@ -1808,11 +1808,21 @@ public struct SwiftifyImportProtocolMacro: ExtensionMacro {
       }
       let overloads = try arguments.map {
         let (method, args) = try parseProtocolMacroParam($0, methods: methods)
-        let function = try constructOverloadFunction(
+        let hasVisibilityModifier = method.modifiers.contains { modifier in
+          let modName = modifier.name.trimmed.text
+          return modName == "public" || modName == "internal" || modName == "open"
+            || modName == "private" || modName == "filePrivate"
+        }
```

Comment:
Not sure if the scenario is reachable, but you're missing `package`, and `filePrivate` is spelled all lowercase. I think you can simplify this to:

```suggestion
        let hasVisibilityModifier = method.modifiers.contains {
          return switch $0.name.trimmed.text {
          case "open", "public", "package", "internal", "fileprivate", "private": true
          default: false
          }
        }
```